_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
```

Comment:
I'll remove 'version'.  I think this is a hold over from how I think about repos but it is probably not a good term to use for learners.